context/app/routes_auth.py

```diff
@@ -83,13 +83,22 @@ def login():
     token_object = tokens.by_resource_server['nexus.api.globus.org']
     nexus_token = token_object['access_token']
 
+    auth_token_object = tokens.by_resource_server['auth.globus.org']
+    auth_token = auth_token_object['access_token']
+
+    user_info_request_headers = {'Authorization': 'Bearer ' + auth_token}
+    user_info = requests.get('https://auth.globus.org/v2/oauth2/userinfo',
+                             headers=user_info_request_headers).json()
+    user_email = user_info['email'] if 'email' in user_info else 'User'
```

Comment:
A little uncomfortable here, and with main.py: 
- I think showing "User" when we don't have an user is a UI choice and I'd like to keep it out of the python.
- If the design changes, we'd need update both of these.
- Muddled semantics: `User` isn't an email, and we would never construct something like `mailto:User`.